site/js/event.js

```diff
@@ -6,6 +6,18 @@ let today_date = date.getDate();
 let today_year = date.getFullYear();
 let curr_hour = date.getHours();
 
+
+/* Read a json file that contains data about Events */
+fetch('/events.json')
+.then((data)=> data.json())
+.then((data)=>{
+   data.forEach(elt => {
+
+    const container = document.createElement('div');  
+    const title = document.createElement('a');
+    const status = document.createElement('p');
+    const guest = document.createElement('p');
+
```

Comment:
why are we fetching the same thing twice?

Reply:
I think he is creating element.